tests/features/support/hooks.js

```diff
@@ -0,0 +1,62 @@
+var {defineSupportCode} = require('cucumber');
+
+var webdriver = require("selenium-webdriver");
+var browserstack = require("browserstack-local");
+
+var createBrowserStackSession = function(config, caps){
+  return new webdriver.Builder().
+    usingServer('http://'+config.server+'/wd/hub').
+    withCapabilities(caps).
+    build();
+}
+
+var config = {
+    server: 'hub-cloud.browserstack.com',
+    capabilities: [
+        {
+            browserName: 'chrome',
+            build: process.env.CIRCLE_BUILD_NUM,
+            project: process.env.CIRCLE_PROJECT_REPONAME + '-' + process.env.CIRCLE_BRANCH,
+            'browserstack.local': true
+        }
+    ]
+};
+
+var username = process.env.BS_USERNAME || config.user;
+var accessKey = process.env.BS_AUTHKEY || config.key;
+
+defineSupportCode(function({After, Before}) {

```

Comment:
__[jshint]__
* 'destructuring binding' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

<!-- Comment by [SideCI](https://sideci.com) -->